@xen-orchestra/web/src/components/host/dashboard/HostDashboardLoadAverageChart.vue

```diff
@@ -45,7 +45,7 @@ const loadAverage = computed<LinearChartData>(() => {
 
     result.set(timestamp, {
       timestamp,
-      value: Number(load.toFixed(2)),
+      value: load ? Number(load.toFixed(2)) : '',
```

Comment:
for type of value, return an number
```suggestion
      value: Number(load.toFixed(2)) || 0,
```
or
```suggestion
      value: load ? Number(load.toFixed(2)) : 0,
```